iac/compositions/cat-full/service_buyer_ui.tf

```diff
@@ -8,18 +74,39 @@ module "buyer_ui_task" {
     http = {
       cpu                   = var.task_container_configs.buyer_ui.http_cpu
       environment_variables = [
-        # TODO set up env vars
+        { name = "AGREEMENTS_SERVICE_API_URL", value = var.buyer_ui_environment["agreements-service-api-url"] },
+        { name = "AUTH_SERVER_BASE_URL", value = var.buyer_ui_environment["auth-server-base-url"] },
+        { name = "AUTH_IDENTITY_BASE_URL", value = var.buyer_ui_environment["auth-identity-base-url"] },
+        { name = "CAT_URL", value = "https://${aws_route53_record.buyer_ui.fqdn}" },
+        { name = "CONCLAVE_WRAPPER_API_BASE_URL", value = var.buyer_ui_environment["conclave-wrapper-api-base-url"] },
+        { name = "DASHBOARD_BANNER", value = var.buyer_ui_environment["dashboard-banner"] },
+        { name = "GCLOUD_INDEX", value = var.buyer_ui_environment["gcloud-index"] },
+        { name = "GCLOUD_SEARCH_API_URL", value = var.buyer_ui_environment["gcloud-search-api-url"] },
+        { name = "GCLOUD_SERVICES_API_URL", value = var.buyer_ui_environment["gcloud-services-api-url"] },
+        { name = "GCLOUD_SUPPLIER_API_URL", value = var.buyer_ui_environment["gcloud-supplier-api-url"] },
+        { name = "GOOGLE_TAG_MANAGER_ID", value = var.buyer_ui_environment["google-tag-manager-id"] },
+        { name = "GOOGLE_SITE_TAG_ID", value = var.buyer_ui_environment["google-site-tag-id"] },
+        { name = "LOGIN_DIRECTOR_URL", value = var.buyer_ui_environment["login-director-url"] },
+        { name = "ROLLBAR_HOST", value = var.buyer_ui_environment["rollbar-host"] },
+        { name = "TENDERS_SERVICE_API_URL", value = "https://${aws_lb.cat_api.dns_name}" },
```

Comment:
Is this using the LB's FQDN to avoid pointing to 'the wrong environment' before a migration has taken place? I take it we don't know if the code validates the CN/SANs of the presented TLS certificate, as if they did they would fail to establish a connection if not using the Route 53 FQDN.

Reply:
But in either case I would always choose a TF reference over a configured value

Reply:
Yep makes sense, just wondered the rationale of the value used over `local.fqdns.cat_api` 👍